index.js

```diff
@@ -384,8 +384,8 @@ Polyglot.prototype.has = function (key) {
 };
 
 // export transformPhrase
-Polyglot.transformPhrase = function transform(phrase, substitutions, locale) {
-  return transformPhrase(phrase, substitutions, locale);
+Polyglot.transformPhrase = function transform(phrase, substitutions, locale, tokenRegex) {
+  return transformPhrase(phrase, substitutions, locale, tokenRegex);
```

Comment:
this was intentionally not provided - see https://github.com/airbnb/polyglot.js/pull/64#discussion_r68440843